libs/api/observability/src/lib/decorators/trace.decorator.ts

```diff
@@ -0,0 +1,14 @@
+import { SetMetadata } from '@nestjs/common';
+
+export const SPAN_ACTIVE = Symbol('SPAN_ACTIVE');
+export const TRACE_NAME = Symbol('TRACE_NAME');
+
+// This will mark the method as a traceable method, meaning it will be proxied with a span. This only works in injected Providers!
```

Comment:
```suggestion
/**
 * This will mark the method as a traceable method, meaning it will be proxied with a span. This only works in injected Providers!
 */
```

Shows the comment in docs as well.